AsyncMediaServerProcessClientApp/asyncMediaServerProcessScript.php

```diff
@@ -126,35 +126,122 @@ function getContentResource ($filepath, KalturaLiveStreamEntry $liveEntry, $work
 		return $resource;
 	}
 	else {
-		try {
-			$client->startMultiRequest();
-			$client->uploadToken->add(new KalturaUploadToken());
+		$client->startMultiRequest();
+		$client->uploadToken->add(new KalturaUploadToken());
 			
-			$client->uploadToken->upload("{1:result:id}", realpath($filepath));
-			$responses = $client->doMultiRequest();
+		$client->uploadToken->upload("{1:result:id}", realpath($filepath));
+		$responses = $client->doMultiRequest();
 			
-			$resource = new KalturaUploadedFileTokenResource();
-			$tokenResponse = $responses[1];
-			if ($tokenResponse instanceof KalturaUploadToken)
-				$resource->token = $tokenResponse->id;
-			else {
-				if ($tokenResponse instanceof Exception) {
-			}
-				logMsg("Content resource creation error: " . $tokenResponse->getMessage() . "]");
-				return null;
+		$resource = new KalturaUploadedFileTokenResource();
+		$tokenResponse = $responses[1];
+		if ($tokenResponse instanceof KalturaUploadToken)
+			$resource->token = $tokenResponse->id;
+		else {
+			if ($tokenResponse instanceof Exception) {
```

Comment:
$tokenResponse will never be instance of Exception. Check if it's array that contains code key.